Servers/UI/OJS.Servers.Ui/Controllers/AccountController.cs

```diff
@@ -83,15 +87,29 @@ public async Task<IActionResult> Login([FromBody]LoginRequestModel model)
             }
             else if (this.webHostEnvironment.IsProduction())
             {
-                return this.Unauthorized(GlobalConstants.ErrorMessages.InactiveLoginSystem);
-            }
+                var signInResult = await this.signInManager
+                    .PasswordSignInAsync(model.UserName, model.Password, model.RememberMe, false);
```

Comment:
What if the user is not admin? We will sign the user anyways?

Reply:
Fixed